lib/connection/messages.coffee

```diff
@@ -75,3 +76,19 @@ module.exports =
       else
         ""
       dismissable: true
+
+  connectExternal: ->
+    tcp.listen().then (port) ->
+      msg = atom.notifications.addInfo "Connect an external process",
+        detail: """
+        To connect a Julia process running in the terminal,
+        run the command:
+
+            using Juno
+            Juno.connect(#{port})
+        """
+        dismissable: true
```

Comment:
What would you think about adding a button that will copy the necessary commands to the clipboard? Shouldn't be too hard, see [here](https://atom.io/docs/api/v1.10.2/NotificationManager#instance-addInfo).